opentelemetry-sdk/tests/metrics/exponential_histogram/test_logarithm_mapping.py

```diff
@@ -0,0 +1,226 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from math import sqrt
+from unittest import TestCase
+
+from opentelemetry.sdk.metrics._internal.exponential_histogram.mapping.errors import (
+    MappingOverflowError,
+    MappingUnderflowError,
+)
+from opentelemetry.sdk.metrics._internal.exponential_histogram.mapping.ieee_754 import (
+    MAX_NORMAL_EXPONENT,
+    MAX_NORMAL_VALUE,
+    MIN_NORMAL_EXPONENT,
+    MIN_NORMAL_VALUE,
+)
+from opentelemetry.sdk.metrics._internal.exponential_histogram.mapping.logarithm_mapping import (
+    LogarithmMapping,
+)
+
+
+def rounded_boundary(scale: int, index: int) -> float:
```

Comment:
what does this function do? i see a func with same name in `opentelemetry-sdk/tests/metrics/exponential_histogram/test_exponent_mapping.py` but they have different impls

Reply:
Those functions calculate the left and right bucket boundaries, I have renamed them as such.